backend/src/app/rpc/commands/teams.clj

```diff
@@ -813,14 +814,15 @@
   [:map {:title "create-team-invitations"}
    [:team-id ::sm/uuid]
    [:role schema:role]
+   [:resend? :boolean]
    [:emails ::sm/set-of-emails]])
 
 (sv/defmethod ::create-team-invitations
   "A rpc call that allow to send a single or multiple invitations to
   join the team."
   {::doc/added "1.17"
    ::sm/params schema:create-team-invitations}
-  [{:keys [::db/pool] :as cfg} {:keys [::rpc/profile-id team-id emails role] :as params}]
+  [{:keys [::db/pool] :as cfg} {:keys [::rpc/profile-id team-id emails role resend?] :as params}]
```

Comment:
We should not use `?` on params on RPC methods, this makes them less usable from JS or other laguanges